.github/workflows/build-test-release.yml

```diff
@@ -150,36 +150,70 @@ jobs:
       fail-fast: false
       matrix:
         splunk-version: ["8.1","8.2"]
+    env:
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Setup kubernetes cluster
+        id: kubernetes
+        run: |
+          ./k3s.sh
+          kubectl get no
+      - name: Setup Splunk-Operator at cluster-scope
+        run: |
+          wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
+          kubectl apply -f splunk-operator.yaml
+          sleep 30
+          kubectl get ns
+          kubectl get pods -A
       - name: Setup for testing
         run: |
           pip install git+https://github.com/pixelb/crudini
           mkdir test-results-${{ matrix.splunk-version }}
+          pip install https://download.splunk.com/misc/packaging-toolkit/splunk-packaging-toolkit-1.0.1.tar.gz
+          ./spl_generation.sh
       - name: Splunk Up
         run: |
-          export SPLUNK_APP_PACKAGE=./tests/e2e/addons/TA_fiction
-          export SPLUNK_ADDON=TA_fiction
-          export SPLUNK_APP_ID=TA_fiction
-          ls -l deps/build/addonfactory_test_matrix_splunk/splunk_matrix.conf
-          export SPLUNK_VERSION=$(crudini --get deps/build/addonfactory_test_matrix_splunk/splunk_matrix.conf ${{ matrix.splunk-version }} VERSION)
+          sudo apt-get install -y curl jq
+          export NAMESPACE_NAME=ta-fiction-external
+          export SPLUNK_ADDON=TA_fiction-0.0.0-dev.spl
+          export SPLUNK_VERSION=${{ matrix.splunk-version }}
+          export SC4S_INDEX_URL=$( curl -s https://api.github.com/repos/splunk/splunk-configurations-base-indexes/releases/latest \ | jq -r '.assets[] | select((.name | contains("spl")) and (.name | contains("search_head") | not) and (.name | contains("indexers") | not ) and (.name | contains("forwarders") | not)) | .browser_download_url ')
```

Comment:
Does this change mean that we don't need `addonfactory_test_matrix_splunk` in `deps/build` folder anymore?

Reply:
Yes, because same as addons we will be fetching the splunk_version from the matrix action: `splunk/addonfactory-test-matrix-action`.